src/pattern-library/components/patterns/transition/SliderPage.tsx

```diff
@@ -76,17 +77,17 @@ export default function SliderPage() {
         </Library.Pattern>
 
         <Library.Pattern title="Props">
-          <Library.Example title="visible">
-            This mandatory boolean prop tells if the Slider is currently
-            displayed or hidden.
+          <Library.Example title="direction">
+            This prop tells if the Slider is currently expanded (<code>in</code>
+            ) or collapsed (<code>out</code>). It is <code>in</code> by default.
```

Comment:
A task for me, especially now that we have reliable routing and fragment navigation, is to create a better centralized documentation for common component props. That is, this documentation probably shouldn't be necessary specifically for `Slider`, instead, we should be able to reuse or link to generalized documentation for this prop.